express-post-json/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const app = express();
+
+let nextId = 1;
+
+const grades = {};
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Listening on port 3000');
+});
```

Comment:
Although it works here it is best to put app.listen at the end of the file as the last thing. That is the common practice.